config/rbac/role.yaml

```diff
@@ -116,33 +167,40 @@ rules:
   - update
   - watch
 - apiGroups:
-  - anywhere.eks.amazonaws.com
+  - apps
   resources:
-  - oidcconfigs
+  - daemonsets
+  - daemonsets/status
+  - deployments
```

Comment:
do we need this in all namespaces? or can we move it to the role and have the permissions namespaced?